src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -476,6 +476,43 @@ class FunSpecTest {
     assertThat(funSpec.toBuilder().build()).isEqualTo(funSpec);
   }
 
+  @Test fun receiverWithKdoc() {
+    val funSpec = FunSpec.builder("toBar")
+        .receiver(String::class, "the string to transform.")
```

Comment:
I think we want to promote using named args for this parameter, can you please change it to `kdoc = "the string to transform."`)